f2/src/forms/LocationInfoForm.js

```diff
@@ -62,71 +28,56 @@ export const LocationInfoForm = (props) => {
-        }) => (
-          <Stack
-            as="form"
-            onSubmit={handleSubmit}
-            shouldWrapChildren
-            spacing={6}
-          >
-            {submitFailed && hasValidationErrors ? (
-              <ErrorSummary>
-                <Trans id="locationinfoPage.hasValidationErrors" />
-              </ErrorSummary>
-            ) : null}
-            <Flex direction="row" align="center" wrap="wrap" mb={10}>
-              <P w="100%">
-                <Trans id="locationinfoPage.skipInfo" />
+      <Formik
+        initialValues={locationInfo}
+        validationSchema={LocationInfoFormSchema()}
+        onSubmit={(values) => {
+          props.onSubmit(values)
+        }}
+      >
+        {({ handleSubmit, handleChange, handleBlur }) => (
+          <Form onSubmit={handleSubmit}>
+            <Container>
+              <Row>
+                <P w="100%">
+                  <Trans id="locationinfoPage.skipInfo" />
+                </P>
+              </Row>
+              <Row>
+                <SkipButton
+                  label={<Trans id="locationinfoPage.skipButton" />}
+                  to="/contactinfo"
+                />
+              </Row>
+              <P>
+                <br />
               </P>
-              <Button
-                as={ReactRouterLink}
-                fontSize={{ base: 'lg', md: 'xl' }}
-                color="black"
-                variant="solid"
-                variantColor="gray"
-                bg="gray.400"
-                borderColor="gray.500"
-                to="/contactinfo"
-                textAlign="center"
-                className="button-link black-button-link"
-              >
-                <Trans id="locationinfoPage.skipButton" />
-                <Icon
-                  focusable="false"
-                  ml={2}
-                  mr={-2}
-                  name="chevron-right"
-                  size="28px"
+              <Row>
+                <Field
+                  name="postalCode"
+                  label={<Trans id="locationinfoPage.postalCode" />}
+                  component={Input}
+                  onChange={handleChange}
+                  onBlur={handleBlur}
+                  id="postalCode"
+                  type="text"
+                  helpText="For example: K1A 0R2"
                 />
-              </Button>
-            </Flex>
-            <Field
-              name="postalCode"
-              label={<Trans id="locationinfoPage.postalCode" />}
-              errorMessage={<Trans id="locationInfoForm.Warning" />}
-              helperText={
-                <Trans id="locationinfoPage.postalCodeExample"></Trans>
-              }
-              component={Input}
-            />
-
-            <NextAndCancelButtons
-              next={<Trans id="locationinfoPage.nextPage" />}
-              button={<Trans id="locationinfoPage.nextButton" />}
-            />
-          </Stack>
+              </Row>
+              <Row>
```

Comment:
For consistency purposes we should use the ErrorMessage above the input field. 
Also, try adding it without the surrounding <Row> component, not sure how it would look but it feels like there is too much space as is.

Reply:
Yes, if no  <Row> component surrounding it, there is too much space before the error message. as following:
<img width="634" alt="Screen Shot 2020-08-28 at 1 47 01 PM" src="https://user-images.githubusercontent.com/51336707/91599875-f834c300-e934-11ea-88eb-d461abf5a678.png">